F50/F50.csproj

```diff
@@ -0,0 +1,62 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.0;netstandard2.0;Xamarin.iOS10;MonoAndroid80;uap10.0.16299</TargetFrameworks>
+    <AssemblyName>Xamarin.F50</AssemblyName>
+    <RootNamespace>Xamarin.F50</RootNamespace>
+    <PackageId>Xamarin.F50</PackageId>
+    <PackageIconUrl>http://xamarin.com/content/images/nuget/xamarin.png</PackageIconUrl>
+    <Summary>F50</Summary>
+    <PackageTags>xamarin, windows, ios, android, xamarin.forms, toolkit, f50</PackageTags>
+    <Title>F50</Title>
+    <Description>F50</Description>
+    <Product>$(AssemblyName) ($(TargetFramework))</Product>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <AssemblyFileVersion>1.0.0.0</AssemblyFileVersion>
+    <Version>1.0.0.0</Version>
+    <PackageVersion>1.0.0.0</PackageVersion>
+    <Authors>Xamarin Inc.</Authors>
+    <PackOnBuild>true</PackOnBuild>
+    <Owners>microsoft,Xamarin,XamarinNuGet</Owners>
+    <NeutralLanguage>en</NeutralLanguage>
+    <Copyright>Copyright 2018</Copyright>
+    <RepositoryUrl>https://github.com/xamarin/F50</RepositoryUrl>
+    <PackageReleaseNotes>See: https://github.com/xamarin/F50 </PackageReleaseNotes>
+    <LangVersion>default</LangVersion>
+    <DefineConstants>$(DefineConstants);</DefineConstants>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GenerateDocumentationFile Condition=" '$(Configuration)' == 'Release' ">true</GenerateDocumentationFile>
+    <UseFullSemVerForNuGet>false</UseFullSemVerForNuGet>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <PackageLicenseUrl>https://github.com/xamarin/F50/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/xamarin/XamarF50inCommunityToolkit</PackageProjectUrl>
```

Comment:
url is 404, did you mean the github repo?

Reply:
Fixed!